entrypoint.sh

```diff
@@ -1,62 +1,40 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Is bash necessary here? We're adding another dependency here if we ever want to slim down the image even more.